central/views/imagecve/db_response.go

```diff
@@ -52,7 +52,7 @@ func (c *imageCVECoreResponse) GetAffectedImageCount() int {
 }
 
 func (c *imageCVECoreResponse) GetFirstDiscoveredInSystem() time.Time {
-	return c.FirstDiscoveredInSystem
+	return *c.FirstDiscoveredInSystem
```

Comment:
What if it's null?

Reply:
good catch. Addressed by changing the signature to return a pointer to give more power to the caller in the decision making.